config/locales/en.yml

```diff
@@ -565,6 +566,7 @@ en:
       late_team: Business unit responsible for late response
       name: Requester
       other_subject_ids: "Police national computer number"
+      criminal_record_reference_number: "Criminal records reference number"
```

Comment:
Maybe a typo?

```suggestion
      criminal_record_reference_number: "Criminal record reference number"
```

Reply:
It was 'record' on the prototype so I left it as that: https://cts-prototype.herokuapp.com/create-case1#width-hint-hint

Happy to change though if it should be 'records' 👍

Reply:
It was 'record' on the prototype so I left it as that: https://cts-prototype.herokuapp.com/create-case1

Reply:
Other way round :) - it's records presently, looks like it should be record

Reply:
So after a few conversations with Alex it turns out it should be Case reference number - so I have changed the PR to reflect this 👍